src/main/java/com/epam/msu/exception/CertificateNotFoundException.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.msu.exception;
+
+import com.epam.msu.entity.Message;
+import com.mysql.cj.Messages;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import java.util.Locale;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class CertificateNotFoundException extends Exception {
```

Comment:
As I saw you have handler for your exceptions, so there is the reason to extends them from RuntimeException and not to write "throws" everywhere